includes/editor-templates/panel-elements.php

```diff
@@ -17,20 +20,32 @@
 
 <script type="text/template" id="tmpl-elementor-panel-categories">
 	<div id="elementor-panel-categories"></div>
-
 	<?php
-	$get_pro_details = apply_filters( 'elementor/editor/panel/get_pro_details', [
+	$get_pro_details = [
 		'link' => 'https://go.elementor.com/pro-widgets/',
 		'message' => __( 'Get more with Elementor Pro', 'elementor' ),
 		'button_text' => __( 'Upgrade Now', 'elementor' ),
-	] );
+	];
+	$promotion_data_sticky = [
+		'url' => 'https://go.elementor.com/go-pro-sticky-widget-panel/',
+		'message' => __( 'Access all Pro widgets. ', 'elementor' ),
+		'button_text' => __( 'Upgrade Now', 'elementor' ),
+	];
+	$get_pro_details = Filtered_Promotions_Manager::get_filtered_promotion_data( $get_pro_details, 'elementor/editor/panel/get_pro_details', 'link' );
```

Comment:
`$get_pro_details `- this one cannot go through the filter. Only put yours through a filter.